addons/l10n_mx/models/res_bank.py

```diff
@@ -19,3 +25,10 @@ class ResPartnerBank(models.Model):
     l10n_mx_edi_clabe = fields.Char(
         "CLABE", help="Standardized banking cipher for Mexico. More info "
         "wikipedia.org/wiki/CLABE")
+    fiscal_country_codes = fields.Char(compute="_compute_fiscal_country_codes")
+
+    @api.depends_context('allowed_company_ids')
+    def _compute_fiscal_country_codes(self):
+        for record in self:
+            allowed_companies = record.company_id or self.env.companies
```

Comment:
Is that right? Do we want it to be visible if the partner is mexican, even if you're in a Belgian company?
Shouldn't we only care about `self.env.companies`?

Reply:
I've tested it and it seems it works as intended, if the companies is anything other that mexican it will not be displayed. it seems that the company_id of the record (res.partner.bank) is empty though which is weird, i will just let self.env.companies to be sure :+1:

Reply:
Yeah, you're right. It's dependent on the company field of the partner.
But you won't be seeing the ResPartnerBank if the company of the partner is not in your allowed_company_ids.
Both are okay I guess then